docs/HACKING.md

```diff
@@ -29,6 +29,9 @@ If you already have a configured cluster, move on to [Build](#build)
 
 ## Build
 
+### Prerequisites
+- [install operator-sdk version 1.15.0 on your machine](https://sdk.operatorframework.io/docs/installation/).
```

Comment:
the SHA of this commit  on my fork is the same as the one on openshift because it's the same commit. GitHub just picked my fork to link to instead. 

https://github.com/openshift/windows-machine-config-operator/commit/bd70beef7c6e80f2da1dac5ac1e5bb10b343d730
https://github.com/wgahnagl/windows-machine-config-operator/commit/bd70beef7c6e80f2da1dac5ac1e5bb10b343d730

Also, that's not the way the rest of the document is written when the links are added in a list. I think we should keep the whole thing as a link.

Reply:
OK, I won't bikeshed on this but please proper case install.